### PR TITLE
Removes an unneeded API call, cleans up JS

### PR DESCRIPTION
**Pull request by abea**

### CODE REVIEW

File: bin/apostrophe [hunk 1]
```diff
@@ -7,46 +7,36 @@ var util = require('../lib/util');
...
-    }
+    output += 'apostrophe v' + aposVersion.trim() + ' (latest) will be installed with a new project, unless the boilerplate project\'s dependencies specify otherwise.';
   }
 
   return output;
 }
 
 util.checkDependencies();
 
-var createCmd   = require('../lib/commands/create')(program);
```
Comment (abea) [inline bin/apostrophe:45]: unused vars.

File: package.json [hunk 2]
```diff
@@ -18,8 +18,7 @@
     "lodash": "^4.17.15",
     "pkginfo": "^0.4.1",
```
Comment (boutell) [inline package.json:19]: Can this go away?

File: bin/apostrophe [hunk 3]
```diff
@@ -7,46 +7,36 @@ var util = require('../lib/util');
 var fs = require('fs');
 
 program.version(getVersion());
-
 // Get version of apostrophe-cli and apostrophe.
 function getVersion() {
-  var pkginfo = require('pkginfo')(module, 'version');
+  require('pkginfo')(module, 'version');
```
Comment (boutell) [inline bin/apostrophe:12]: Why is this still here? Isn't this what crashes?

File: package.json (hunk 2 above)
Comment (abea) [reply package.json:19]: We're using it, but not much. We could use the `exports.version = require('/path/to/package.json').version;` method that pkginfo decries.

File: bin/apostrophe (hunk 3 above)
Comment (abea) [reply bin/apostrophe:12]: No, this is looking at the installed CLI module. But it's easy to replace.

**Pull request merged**